.github/workflows/stage_release.yml

```diff
@@ -0,0 +1,126 @@
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements. See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License. You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+name: Stage Release
+
+# This workflow will stage a kroxylicious junit extension release.  The release artefacts will be
+# built, signed, and staged on Nexus.  The staging repository will be closed.  A PR will be opened
+# containing commits that versions the release and reopen main for development at the next snapshot version.
+#
+# Once the staged artefacts have been verified, run the workflow deploy_release with the argument `release` to
+# release the artefacts to Maven Central and merge the release PR.
+#
+# It requires the following:
+# variables:
+# KROXYLICIOUS_SONAR_TYPE_TOKEN_USERNAME - Sonar Type Access User Token Username
+# secrets:
+# KROXYLICIOUS_RELEASE_TOKEN             - GitHub PAT wih content/createPullRequest permission for Kroxylicious repo.
+# KROXYLICIOUS_RELEASE_GPG_PRIVATE_KEY   - GPG private key for Kroxylicious
```

Comment:
`GPG private key for Kroxylicious` sounds like it contradicts the description of which GPG Key to use from the PR description. 

I presume this should really be the Private key of the project admin conducting the release.